src/lesson02/Task02.java

```diff
@@ -0,0 +1,22 @@
+package lesson02;
+import java.util.Scanner;
+
+public class Task02 {
+    public static void main(String[] args){
+        /*int w = 2;
+        int h = 4;
+        System.out.println("Perimeter:" + 2*(w+h));
+        System.out.println("Square:" + w*h);*/
+        double w;
+        double h;
+        Scanner in = new Scanner(System.in);
+        System.out.println("Введите ширину:");
+        w = Double.parseDouble(in.nextLine());
+        System.out.println("Введите высоту:");
+        h = Double.parseDouble(in.nextLine());
+        System.out.println("Вывести периметр:" + 2*(w+h));
+        System.out.println("Вывести площадь:" + w*h);
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"